src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicator.java

```diff
@@ -297,6 +550,820 @@ private void populateControlCapabilities(Map<String, String> stats, List<Advance
 		}
 	}
 
+	//region Control device
+
+	/**
+	 * This method is used to control power on
+	 */
+	public void powerOn() {
```

Comment:
powerOn and powerOff are sharing a similar logic. I think we can move it to the generic method